src/axom/quest/ArrayIndexer.hpp

```diff
@@ -10,8 +10,19 @@
 #include "axom/core/StackArray.hpp"
 #include "axom/core/numerics/matvecops.hpp"
 
+#include <iostream>
+#include <sstream>
+
 namespace axom
 {
+struct ArrayStrideOrder
+{
+  static constexpr int ARBITRARY = 0;        // Neither row nor column
+  static constexpr int ROW = 1;              // Row-major
+  static constexpr int COLUMN = 2;           // Column-major
+  static constexpr int BOTH = ROW | COLUMN;  // 1D arrays are both
+};
```

Comment:
Enum was how I first coded it but I couldn't do bit-wise operations on the enums.  I guess it's because the results aren't guaranteed to be a defined enum value.

Reply:
I'll make it an enum and convert to ints as needed to do bitwise operations.